src/interpolations.jl

```diff
@@ -115,7 +128,7 @@ getnbasefunctions(::Lagrange{1, RefCube, 1}) = 2
 getnfacenodes(::Lagrange{1, RefCube, 1}) = 1
 getfacelist(::Type{Lagrange{1, RefCube, 1}}) = ((1,),(2,))
 
-function value!(ip::Lagrange{1, RefCube, 1}, N::AbstractVector, ξ::Vec{1})
+function value!(ip::Lagrange{1, RefCube, 1}, N::AbstractVector, ξ::AbstractVector)
```

Comment:
Right. If we called `value!` for each independent shape function we could have used AD from `Tensors`, right?